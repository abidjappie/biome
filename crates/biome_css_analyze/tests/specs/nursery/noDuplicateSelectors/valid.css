.foo {
    width: 12;
}

@media (min-width: 10px) {
    .foo {
        width: 12;
    }
}

/* 
.foo {
    .foo {
        width: 12;
    }
}

.foo {
    width: 12;
}

.bar {
    width: 12;
}

.foo .bar {
    width: 12;
}

.bar .foo {
    width: 12;
}

a b {
    width: 12;
}

a {

    & b,
    & c {
        width: 12;
    }
} */