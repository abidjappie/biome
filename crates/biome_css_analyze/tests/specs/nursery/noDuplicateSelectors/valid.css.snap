---
source: crates/biome_css_analyze/tests/spec_tests.rs
expression: valid.css
---
# Input
```css
/* selectors using parts of other selectors */
.foo .bar {
    div {
        
    }
    s {
        t {

        }
        u {
            
        }
    }
}
.qoo {}
.qar {}
.sar .soo {}
```
